examples/online-application-portal/Ballerina online application portal.md

```diff
@@ -0,0 +1,34 @@
+# Ballerina online application portal
```

Comment:
Should this be
```suggestion
# Guidewire online application portal
```
?

Reply:
I think it would be better to keep `Ballerina` here as this is a Ballerina application. The details regarding the underlying technology is described in the description.